package.json

```diff
@@ -63,5 +64,14 @@
     ],
     "all": true,
     "check-leaks": true
+  },
+  "mocha": {
```

Comment:
Can you move mocha configuration into a `.mocharc.js` file?

Here's the example Mocha config JS file:
https://github.com/mochajs/mocha/blob/master/example/config/.mocharc.js

And here's the example YML file with every configuration option:
https://github.com/mochajs/mocha/blob/master/example/config/.mocharc.yml